.cirrus.yml

```diff
@@ -14,6 +14,11 @@ env:
   ARTIFACTORY_ACCESS_TOKEN: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-private-reader access_token]
   DEVELOCITY_TOKEN: VAULT[development/kv/data/develocity data.token]
   DEVELOCITY_ACCESS_KEY: develocity.sonar.build=${DEVELOCITY_TOKEN}
+  # analysis on next
+  SONAR_HOST_URL: "https://next.sonarqube.com/sonarqube"
+  SONAR_TOKEN: VAULT[development/kv/data/next data.token]
+  SONAR_PROJECT_KEY: "org.sonarsource.xml:xml"
```

Comment:
The token should be moved back closer to usage to avoid having it spill into other tasks

Reply:
Meaning that we can duplicate the information for both the build tasks and the matrix of shadow analyses